src/QMCWaveFunctions/Jastrow/J1OrbitalSoA.h

```diff
@@ -59,11 +71,19 @@ struct J1OrbitalSoA : public WaveFunctionComponent
   aligned_vector<int> DistIndice;
   Vector<posT> Grad;
   Vector<valT> Lap;
-  ///Container for \f$F[ig*NumGroups+jg]\f$
+  ///Container for \f$F[ig*NIons+jg]\f$
   std::vector<FT*> F;
+  ///container for the unique Jastrow functions
+  std::vector<std::unique_ptr<FT>> J1Unique;
```

Comment:
J1Unique replace the old role of F. However, F was redefined. For this reason, all the F in the the derived class OneBodyJastrowOrbitalBspline.cpp should be updated.